AllReadyApp/Web-App/AllReady.UnitTest/Features/Manage/SendAccountSecurityTokenSmsHandlerTests.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Threading.Tasks;
+using AllReady.Features.Manage;
+using AllReady.Services;
+using Moq;
+using Xunit;
+
+namespace AllReady.UnitTest.Features.Manage
+{
+    public class SendAccountSecurityTokenSmsHandlerTests
+    {
+        [Fact]
+        public async Task SendAccountSecurityTokenSmsHandlerInvokesSendSmsAsyncWithCorrectPhoneNumberAndMessage()
+        {
+            var message = new SendAccountSecurityTokenSms { PhoneNumber = "phoneNumber", Token = "code" };
+            var smsMessage = $"Your allReady account security code is: {message.Token}";
+
+            var smsSender = new Mock<ISmsSender>();
+            var sut = new SendAccountSecurityTokenSmsHandler(smsSender.Object);
+            await sut.Handle(message);
+
+            smsSender.Verify(x => x.SendSmsAsync(message.PhoneNumber, smsMessage));
```

Comment:
I wouldn't actually worry about the explicit `smsMessage` in this case, rather just that the expected invocation had occurred. I'm thinking of two things in particular: 

 1. change of the message in the code would require a change to the unit test (which may or may not add much value)
 1. it would be difficult to make this test work as-is factoring in globalization/localiztion

Thoughts?